image_client/botany_importer.py

```diff
@@ -23,20 +20,21 @@
 # CAS0410512_a
 
 class BotanyImporter(Importer):
-    def __init__(self):
-        self.logger = logging.getLogger('Client.BotanyImporter')
-        super().__init__(botany_importer_config,"Botany")
 
+
+    def __init__(self, paths):
+        self.logger = logging.getLogger('Client.BotanyImporter')
+        super().__init__(paths, "Botany")
```

Comment:
It looks like we never used the previous botany_importer_config argument and we don't use "paths" now (in Importer.py). Go ahead and remove that argument here, in inporter.py, and in any other class that imports importer.py